cart/render-line-items.js

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
As I commented in your canvas, I think it is good practice to separate your functions into another file. You could also have a 'cart-utils.js' that contains all functions you use on the cart page specifically. There is no right answer though, just the best way you can stay organized :)